xo.py

```diff
@@ -0,0 +1,180 @@
+import random
+import string
+
+ALL_LETTERS = string.ascii_lowercase
+DEFAULT_BOARD_SIZE = 3
+
+
+def start_game(board_size=DEFAULT_BOARD_SIZE):
+
+    players = gather_players_names()
+
+    while True:
+        # Randomize play order.
+        random.shuffle(players)
+
+        print('First player to start is: ' + players[0])
+
+        main_game_logic(players, board_size)
+
+        start_another = __get_console_input('Another game? (Y to restart)', 1, 1)
+        if start_another.lower() != 'y':
+            break
+
+
+def gather_players_names():
+    player_1_name = __get_console_input('Player1, what\'s your name?', 3, 20)
+    player_2_name = __get_console_input('Player2, what\'s your name?', 3, 20)
+
+    return [player_1_name, player_2_name]
```

Comment:
No need for a list here, you can use a tuple:
`return player_1_name, player_2_name`